### PR TITLE
Added new classes for common actions #823 

### PR DESCRIPTION
**Pull request by breiler**
Added new classes for the common action as discussed in #823

### CODE REVIEW

**Comment by winder** [comment]
Thank you! I've been out of town, taking a look at this soon.

File: ugs-core/src/com/willwinder/universalgcodesender/GrblController.java [hunk 1]
```diff
@@ -669,9 +669,9 @@ else if ((this.controllerStatus.getState().equals("Hold")
             if (zLocationMM > this.maxZLocationMM) {
                 maxZLocationMM = zLocationMM;
             }
-        }
 
-        dispatchStatusString(controllerStatus);
+            dispatchStatusString(controllerStatus);
```
Comment (winder) [inline ugs-core/src/com/willwinder/universalgcodesender/GrblController.java]: Was this change intentional?

Comment (breiler) [reply ugs-core/src/com/willwinder/universalgcodesender/GrblController.java]: It was not my intention to commit it, but I did revert it in the next commit.

File: ugs-platform/branding/src/main/nbm-branding/core/core.jar/org/netbeans/core/ui/Bundle.properties [hunk 2]
```diff
@@ -1 +1,24 @@
-LBL_Copyright=Universal G-Code Sender documentation can be found at <a href="http://winder.github.io/ugs_website/">here</a>.<br/>Bugs should be reported on github <a href="https://github.com/winder/Universal-G-Code-Sender/issues">github.com/winder/Universal-G-Code-Sender</a><br/><br/>Pendant icon made by Dario Ferrando from www.flaticon.com is licensed by CC 3.0 BY
```
Comment (winder) [inline ugs-platform/branding/src/main/nbm-branding/core/core.jar/org/netbeans/core/ui/Bundle.properties]: Thanks for this improvement too!
